test/test_ssl.rb

```diff
@@ -429,6 +458,15 @@ def test_timeout
 
 private
 
+  def set_x509_const(name, value)
```

Comment:
I'd use a block here and `ensure` after `yield`

Reply:
👍